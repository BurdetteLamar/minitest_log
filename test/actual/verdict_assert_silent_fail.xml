<log>
  <summary_ verdicts='1' failures='1' errors='0'/>
  <verdict_ method='verdict_assert_silent?' outcome='failed' id='1'>
    <exception_ class='Minitest::Assertion' message='In stderr.'>
      <backtrace_/>
    </exception_>
  </verdict_>
</log>
