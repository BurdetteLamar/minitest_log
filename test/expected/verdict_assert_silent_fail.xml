<log>
  <summary_ verdicts='2' failures='1' errors='0'/>
  <verdict_ method='verdict_assert_silent?' outcome='failed' id='1'>
    <exception_ class='Minitest::Assertion' message='In stderr.'>
      <backtrace_>
      </backtrace_>
    </exception_>
  </verdict_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
